"rectype","issueid","project_owner","project_name","actor","time","text","action","title"
"issue_title","1388","nilearn","nilearn","chrisfilo","2017-02-23 22:28:12","```
/home/ubuntu/nilearn/nilearn/image/resampling.py:472: UserWarning: Casting data from int64 to float64
  warnings.warn(""Casting data from %s to %s"" % (data.dtype.name, aux))
/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/scipy/ndimage/interpolation.py:430: UserWarning: The behaviour of affine_transform with a one-dimensional array supplied for the matrix parameter has changed in scipy 0.18.0.
  ""The behaviour of affine_transform with a one-dimensional ""
/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/sklearn/utils/validation.py:526: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)
/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/numpy/lib/function_base.py:3858: RuntimeWarning: Invalid value encountered in median
  r = func(a, **kwargs)

Traceback (most recent call last):
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/cmdline.py"", line 296, in main
    app.build(opts.force_all, filenames)
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/application.py"", line 333, in build
    self.builder.build_update()
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py"", line 251, in build_update
    'out of date' % len(to_build))
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py"", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py"", line 360, in write
    self._write_serial(sorted(docnames), warnings)
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py"", line 370, in _write_serial
    self.warn(*warning, **kwargs)
  File ""/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/application.py"", line 399, in warn
    raise SphinxWarning(warntext)
sphinx.errors.SphinxWarning: /home/ubuntu/nilearn/doc/auto_examples/01_plotting/plot_surf_stat_map.rst:28: WARNING: unknown document: plot_seed_to_voxel_correlation


Warning, treated as error:
/home/ubuntu/nilearn/doc/auto_examples/01_plotting/plot_surf_stat_map.rst:28: WARNING: unknown document: plot_seed_to_voxel_correlation
```

More details: https://circleci.com/gh/nilearn/nilearn/2276 I believe this was introduced in #1387.","start issue","unknown document: plot_seed_to_voxel_correlation"
"issue_closed","1388","nilearn","nilearn","KamalakerDadi","2017-03-02 15:54:15","","closed issue","unknown document: plot_seed_to_voxel_correlation"
"issue_comment","1388","nilearn","nilearn","GaelVaroquaux","2017-02-24 19:04:00","Yes, it would be nice. It vanished a little while ago 

⁣Sent from my phone. Please forgive brevity and mis spelling​

On Feb 24, 2017, 19:59, at 19:59, Chris Filo Gorgolewski <notifications@github.com> wrote:
>Should I turn on CircleCI integration for forks and PRs so such things
>would be caught during PR review?
>
>-- 
>You are receiving this because you modified the open/close state.
>Reply to this email directly or view it on GitHub:
>https://github.com/nilearn/nilearn/issues/1388#issuecomment-282374733
","",""
"issue_comment","1388","nilearn","nilearn","GaelVaroquaux","2017-03-02 15:06:04","That's fixed, isn't it?","",""
"issue_comment","1388","nilearn","nilearn","GaelVaroquaux","2017-03-02 15:50:55","Great. Can you close it, please.
","",""
